templates/admin/actions.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
+
+{% if action_form %}
+  <div class="actions">
+    {% for field in action_form reversed %}
```

Comment:
Note that action will be always at the bottom

Reply:
Not sure I get this one without seeing it :)